internal/controller/pkg/revision/runtime.go

```diff
@@ -273,11 +273,13 @@ func (b *RuntimeManifestBuilder) Service(overrides ...ServiceOverride) *corev1.S
 		ServiceWithSelectors(b.podSelectors()),
 		ServiceWithAdditionalPorts([]corev1.ServicePort{
 			{
+				Name:       webhookPortName,
 				Protocol:   corev1.ProtocolTCP,
 				Port:       servicePort,
 				TargetPort: intstr.FromInt32(servicePort),
 			},
-		}))
+		}),
+		ServiceWithPort(webhookPortName, servicePort))
```

Comment:
might be useful to capture the reason in code comments here, as to why this is being set again.., to avoid any confusion in the future..